--- src/05.Infrastructure/SIASUN.RCS.EntityFrameworkCore/EntityFrameworkCore/RCSEntityFrameworkCoreModule.cs
+++ src/05.Infrastructure/SIASUN.RCS.EntityFrameworkCore/EntityFrameworkCore/RCSEntityFrameworkCoreModule.cs
@@ -56,12 +56,8 @@
         {
             options.PreConfigure<RCSDbContext>(ctx =>
             {
-                var interceptor = ctx.ServiceProvider.GetService<SIASUN.RCS.EntityFrameworkCore.Auditing.EntityAuditInterceptor>();
-                if (interceptor != null)
-                {
-                    ctx.DbContextOptions.AddInterceptors(interceptor);
-                }
-                var interceptor = ctx.ServiceProvider.GetRequiredService<SIASUN.RCS.EntityFrameworkCore.Auditing.EntityAuditInterceptor>();
+                var interceptor = ctx.ServiceProvider.GetRequiredService<SIASUN.RCS.EntityFrameworkCore.Auditing.EntityAuditInterceptor>();
                 ctx.DbContextOptions.AddInterceptors(interceptor);
             });
 
