--- src/03.Application/SIASUN.RCS.Application/Auditing/AuditLogFilterRuleAppService.cs
+++ src/03.Application/SIASUN.RCS.Application/Auditing/AuditLogFilterRuleAppService.cs
@@ -3,6 +3,7 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Authorization;
 using SIASUN.RCS.Permissions;
+using System.Linq;
 using Volo.Abp.Application.Dtos;
 using Volo.Abp.Application.Services;
 using Volo.Abp.Domain.Repositories;
@@ -26,8 +27,11 @@
 
         public async Task<PagedResultDto<AuditLogFilterRuleDto>> GetListAsync(GetAuditLogFilterRulesInput input)
         {
-            var count = await _ruleRepository.GetCountAsync();
-            var list = await _ruleRepository.GetPagedListAsync(input.SkipCount, input.MaxResultCount, "CreationTime DESC");
+            var query = await _ruleRepository.GetQueryableAsync();
+            if (!string.IsNullOrWhiteSpace(input.Filter)) query = query.Where(x => x.Name.Contains(input.Filter) || x.PathPattern.Contains(input.Filter));
+            var count = query.Count();
+            var list = query.OrderByDescending(x => x.CreationTime).Skip(input.SkipCount).Take(input.MaxResultCount).ToList();
+            
             return new PagedResultDto<AuditLogFilterRuleDto>(count, ObjectMapper.Map<List<AuditLogFilterRule>, List<AuditLogFilterRuleDto>>(list));
         }
 
